src/features/HomePage/noticesApi.ts

```diff
@@ -0,0 +1,11 @@
+import mockNotices from './mock.json';
+
+export type Notice = {
+  id: number;
+  message: string;
+};
+
+export const fetchNotices = (): Notice[] => {
+  const data = mockNotices.notices as Notice[];
+  return data;
+};
```

Comment:
The mock data is fine, but I want us to use the rtk-query api. But also a question arises. Where are these notices coming from? We dont have an API implementation for this yet.

Since this is not an existing functionality in the backend, I would not implement this yet. We can talk about this in our next weekly meeting.

Reply:
I remember something talked about the "notification ball" (for new messages), but it was implemented in the last PR? So we are not mixing these two features?